src/Routes.js

```diff
@@ -11,14 +11,17 @@ const routes = {
 };
 
 function getPageUrl(routeId) {
+  if (!routes[routeId]) return undefined;
   return routes[routeId].url;
 }
 
 function getPageSeo(routeId) {
+  if (!routes[routeId]) return undefined;
   return routes[routeId].seo;
 }
 
 function getPageTitle(routeId) {
+  if (!routes[routeId]) return undefined;
   return routes[routeId].seo.title;
```

Comment:
If you feel like writing these functions more concisely, you can do something like:
`return routes[routeId] && routes[routeId].seo && routes[routeId].seo.title` - this will short circuit and return `undefined` if `routes` has no key `routeId`. There's nothing wrong with having an `if` guard like this though, just a bit more code.